.maintain/runtime-weight-template.hbs

```diff
@@ -73,6 +73,7 @@ mod tests {
   use crate::constants::{MAXIMUM_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO};
   use crate::weights::extrinsic_weights::ExtrinsicBaseWeight;
 
+  #[allow(dead_code)]
   struct BlockWeights;
```

Comment:
This isn't always dead code, but sometimes is and we just don't need generated files throwing lints